src/main/java/org/apache/bcel/util/package.html

```diff
@@ -17,17 +17,13 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
 <html>
 <head>
-<!--
-$Id$
--->
 </head>
 <body bgcolor="white">
 <p>
 This package contains utility classes for the
 <a href="https://commons.apache.org/bcel/">Byte Code Engineering
 Library</a>, namely:
 </p>
-<p>
```

Comment:
Rather than putting these differences in this pull request, it would be nicer to:
 * pull apache fork into codespecs fork
 * pull codespecs fork into typetools fork
Now, this change is not needed, and as a side benefit the forks are more up to date.

Reply:
done